packages/documentation/src/pages/getting-started/index.mdx

```diff
@@ -17,10 +17,20 @@ These instructions cover the installation of dependencies needed for running VA.
 
 1. Install `nvm`:
 
+Mac:
+
 ```bash
 brew update && brew install nvm
 ```
 
+Linux:
+
+To get the latest version of NVM visit the official page. [NVM](https://github.com/nvm-sh/nvm)
```

Comment:
What if we had them install `linuxbrew`- we wouldn't need to change these instructions. We could tweak the prerequisites to link to mac vs [linux](https://docs.brew.sh/Homebrew-on-Linux) installations of brew.

Reply:
I'd rather not introduce more overhead.